react/week3/react_week3/src/FetchedData.js

```diff
@@ -0,0 +1,184 @@
+import React,{useEffect, useState} from 'react';
+import PropTypes from 'prop-types';
+import Timer from './Timer.js';
+
+
+ const toDo_url = 'https://gist.githubusercontent.com/benna100/391eee7a119b50bd2c5960ab51622532/raw';
```

Comment:
Values that never change - `constants` - are usually written in uppercase: `TODO_URL`